ml/rl/workflow/types.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+
+from typing import Dict, List, NamedTuple, Optional
+
+from ml.rl.preprocessing.normalization import (
+    DEFAULT_MAX_QUANTILE_SIZE,
+    DEFAULT_MAX_UNIQUE_ENUM,
+    DEFAULT_NUM_SAMPLES,
+    DEFAULT_QUANTILE_K2_THRESHOLD,
+)
+
+IGNORE_SANITY_CHECK_FAILURE = True
+
+
+class TableSpec(NamedTuple):
+    table: str
```

Comment:
Let's name this `table_url`

Reply:
Right now, it's more of a table_name, not a url. Maybe table_name makes more sense?